package.json

```diff
@@ -1,162 +1,163 @@
-    "concurrently": "^7.2.1",
-    "copy-webpack-plugin": "^11.0.0",
-    "cross-env": "^7.0.3",
-    "css-loader": "^6.7.1",
-    "electron": "^25.3.2",
-    "electron-installer-common": "^0.10.3",
-    "eslint": "^8.16.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-airbnb-typescript": "^17.0.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-import-resolver-typescript": "^2.7.1",
-    "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-prefer-arrow-functions": "^3.1.4",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.30.0",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-memo": "^0.0.3",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "file-loader": "^6.2.0",
-    "image-webpack-loader": "^8.1.0",
-    "jest": "^29.5.0",
-    "node-loader": "^2.0.0",
-    "nodemon": "^2.0.16",
-    "prettier": "^2.8.8",
-    "react-refresh": "^0.13.0",
-    "sass": "^1.54.4",
-    "sass-loader": "^13.0.2",
-    "style-loader": "^3.3.1",
-    "stylelint": "^14.13.0",
-    "stylelint-config-prettier-scss": "^0.0.1",
-    "stylelint-config-standard-scss": "^5.0.0",
-    "stylelint-prettier": "^2.0.0",
-    "ts-jest": "^29.1.0",
-    "ts-node": "^10.9.1",
-    "typescript": "^5.0.4"
-  },
-  "dependencies": {
-    "@babel/plugin-transform-react-jsx": "^7.17.12",
-    "@chainner/navi": "^0.6.0",
-    "@chakra-ui/icons": "^2.0.11",
-    "@chakra-ui/react": "^2.3.5",
-    "@emotion/react": "^11.9.0",
-    "@emotion/styled": "^11.8.1",
-    "@fontsource/open-sans": "^4.5.10",
-    "@octokit/rest": "^19.0.13",
-    "@react-nano/use-event-source": "^0.12.0",
-    "adm-zip": "^0.5.10",
-    "bezier-js": "^6.1.0",
-    "chakra-ui-markdown-renderer": "^4.1.0",
-    "chokidar": "^3.5.3",
-    "decompress": "^4.2.1",
-    "electron-log": "^4.4.8",
-    "electron-squirrel-startup": "^1.0.0",
-    "eventsource": "^2.0.2",
-    "fast-deep-equal": "^3.1.3",
-    "framer-motion": "^6.3.6",
-    "html-to-image": "^1.10.8",
-    "i18next": "^22.0.5",
-    "immer": "^10.0.2",
-    "javascript-time-ago": "^2.5.9",
-    "minisearch": "^6.1.0",
-    "node-localstorage": "^2.2.1",
-    "nodejs-file-downloader": "^4.9.3",
-    "os-utils": "^0.0.14",
-    "patch-package": "^8.0.0",
-    "portfinder": "^1.0.32",
-    "re-resizable": "^6.9.9",
-    "react": "^18.1.0",
-    "react-colorful": "^5.6.1",
-    "react-dom": "^18.1.0",
-    "react-hotkeys-hook": "^3.4.6",
-    "react-i18next": "^12.0.0",
-    "react-icons": "^4.10.1",
-    "react-markdown": "^8.0.3",
-    "react-query": "^3.39.3",
-    "react-time-ago": "^7.2.1",
-    "reactflow": "^11.5.6",
-    "rregex": "^1.7.0",
-    "semver": "^7.3.7",
-    "systeminformation": "^5.11.16",
-    "undici": "^5.22.1",
-    "use-context-selector": "^1.4.0",
-    "use-debounce": "^8.0.1",
-    "uuid": "^8.3.2",
-    "wcag-contrast": "^3.0.0",
-    "yargs": "^17.5.1"
-  },
-  "optionalDependencies": {
-    "appdmg": "^0.6.6"
-  }
+    "name": "chainner",
```

Comment:
can you undo the formatting change? i think if you just do another `npm i` it should revert it to what it wants. if that doesn't do anything, let me know and we can just leave it

Reply:
Done. Had to adjust my JSON formatter.